### ISSUE TITLE
Failed to start mtproto-proxy.service: Unit epmd.service not found.

### ISSUE THREAD

**Issue by KarelWintersky** [seriyps/mtproto_proxy#5]
Ubuntu 16.04, by default erlang 17, so, I installed Erlang from binaries.

Result:

`Failed to start mtproto-proxy.service: Unit epmd.service not found.`

Compete installation log:

```
# cd /opt
# wget https://packages.erlang-solutions.com/erlang/esl-erlang/FLAVOUR_1_general/esl-erlang_21.2.4-1~ubuntu~xenial_amd64.deb 

# dpkg -i esl-erlang_21.2.4-1\~ubuntu\~xenial_amd64.deb

Selecting previously unselected package esl-erlang.
(Reading database ... 165794 files and directories currently installed.)
Preparing to unpack esl-erlang_21.2.4-1~ubuntu~xenial_amd64.deb ...
Unpacking esl-erlang (1:21.2.4-1) ...
dpkg: dependency problems prevent configuration of esl-erlang:
 esl-erlang depends on libwxbase2.8-0 | libwxbase3.0-0 | libwxbase3.0-0v5; however:
  Package libwxbase2.8-0 is not installed.
  Package libwxbase3.0-0 is not installed.
  Package libwxbase3.0-0v5 is not installed.
 esl-erlang depends on libwxgtk2.8-0 | libwxgtk3.0-0 | libwxgtk3.0-0v5; however:
  Package libwxgtk2.8-0 is not installed.
  Package libwxgtk3.0-0 is not installed.
  Package libwxgtk3.0-0v5 is not installed.
 esl-erlang depends on libsctp1; however:
  Package libsctp1 is not installed.

dpkg: error processing package esl-erlang (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 esl-erlang

# apt install -f

Reading package lists... Done
Building dependency tree       
Reading state information... Done
Correcting dependencies... Done
The following additional packages will be installed:
  libdrm-amdgpu1 libdrm-intel1 libdrm-nouveau2 libdrm-radeon1 libgl1-mesa-dri libgl1-mesa-glx libglapi-mesa libllvm6.0 libnotify4
  libpciaccess0 libsctp1 libsensors4 libwxbase3.0-0v5 libwxgtk3.0-0v5 libx11-xcb1 libxcb-dri2-0 libxcb-dri3-0 libxcb-glx0 libxcb-present0
  libxcb-sync1 libxshmfence1 libxxf86vm1
Suggested packages:
  lksctp-tools lm-sensors
Recommended packages:
  libtxc-dxtn-s2tc | libtxc-dxtn-s2tc0 | libtxc-dxtn0 notification-daemon
The following NEW packages will be installed:
  libdrm-amdgpu1 libdrm-intel1 libdrm-nouveau2 libdrm-radeon1 libgl1-mesa-dri libgl1-mesa-glx libglapi-mesa libllvm6.0 libnotify4
  libpciaccess0 libsctp1 libsensors4 libwxbase3.0-0v5 libwxgtk3.0-0v5 libx11-xcb1 libxcb-dri2-0 libxcb-dri3-0 libxcb-glx0 libxcb-present0
  libxcb-sync1 libxshmfence1 libxxf86vm1
0 upgraded, 22 newly installed, 0 to remove and 0 not upgraded.
1 not fully installed or removed.
Need to get 26.1 MB of archives.
After this operation, 226 MB of additional disk space will be used.
Do you want to continue? [Y/n] 
...

# dpkg -i esl-erlang_21.2.4-1\~ubuntu\~xenial_amd64.deb
(Reading database ... 170229 files and directories currently installed.)
Preparing to unpack esl-erlang_21.2.4-1~ubuntu~xenial_amd64.deb ...
Unpacking esl-erlang (1:21.2.4-1) over (1:21.2.4-1) ...
Setting up esl-erlang (1:21.2.4-1) ...

# erl --version
Erlang/OTP 21 [erts-10.2.3] [source] [64-bit] [smp:1:1] [ds:1:1:10] [async-threads:1] [hipe]

Eshell V10.2.3  (abort with ^G)

# git clone https://github.com/seriyps/mtproto_proxy.git
# cd mtproto_proxy/
# cp config/vm.args.example config/prod-vm.args
# cp config/sys.config.example config/prod-sys.config
# nano config/prod-sys.config 

# make && sudo make install

# sudo systemctl start mtproto-proxy

Failed to start mtproto-proxy.service: Unit epmd.service not found.

```

_Originally posted by @KarelWintersky in https://github.com/seriyps/mtproto_proxy/issues/4#issuecomment-458544946_

**Comment by hookzof** [comment]
I just deleted in "/config/mtproto-proxy.service" this line "Requires = epmd.service".
🌚

**Comment by KarelWintersky** [comment]
@hookzof , I know. But this solution must be in original repo :)

**Comment by seriyps** [comment]
The problem is that `Requires = epmd.service` will work better with Ubuntu 18.04 and other distributions where epmd is a separate service. I think I will add extra check that system have a epmd service enabled, and will add it if it's not (I did that on CentOS systems)

**Comment by seriyps** [comment]
Should be fixed by ba5a83119819f4861ba956d433b69af51957d341. But in general I would recomend to upgrade to Ubuntu 18.04
